shopinvader/models/res_partner.py

```diff
@@ -51,14 +51,9 @@ class ResPartner(models.Model):
     )
 
     @api.model
-    def _is_partner_duplicate_allowed(self):
-        """Check if partner duplication is allowed
-
-        This parameter is configured through res.config.settings
-        """
+    def _is_partner_duplicate_prevented(self):
```

Comment:
```suggestion
    def _is_partner_duplicate_not_allowed(self):
```
sounds better, but it's just my taste I guess :)